### DISCUSSION TITLE
Thank you for taking this up

### DISCUSSION THREAD

**Discussion by hsjdc** [powerpaul17/nc_money#40] [General]
As a longtime gnucash and nextcloud user, I'm delighted to learn about this project (and I realize the objective is not to bring all of gnucash to nc_money). Though I am neither a coder nor (professional) accountant, I would be happy to contribute in whatever ways are helpful (on the wiki, e.g., or account structure/db design). A project like this could be among the stars of the platform, particularly for not-for-profit organizations that rely on paid, proprietary accounting systems. 

But mostly just wanted to say thanks.

**Comment by powerpaul17** [comment]
Sorry for the late reply and thanks for the nice words. You can help in whatever way you want, the wiki should be publicly editable and still lacks a lot of information. Maybe we could also host documentation as a github page at some point but that would involve a bit of coding. If you have experience in database design we could have discussions here when something comes up.. :slightly_smiling_face: Also feel free to create issues/bug reports if you find find some..

**Comment by walttheboss** [comment]
I agree completely.  I am a sysadmin but not a coder.  I run a network of small international schools across SE Asia.  We are looking for an accounting software and would love it to be inside of NC.  Next year. Sept 23 to May 24 I  have an IT intern.  He is an IT savant. 19 years old and can fly in about 10 programming languages.  I will contribute 5 hours a week of his time to help with this.  Let me know how to start.
